webhook.go

```diff
@@ -37,7 +37,8 @@ type WebhookParams struct {
 	AllowedMentions *MessageAllowedMentions `json:"allowed_mentions,omitempty"`
 	// Only MessageFlagsSuppressEmbeds and MessageFlagsEphemeral can be set.
 	// MessageFlagsEphemeral can only be set when using Followup Message Create endpoint.
-	Flags MessageFlags `json:"flags,omitempty"`
+	Flags      MessageFlags `json:"flags,omitempty"`
+	ThreadName string       `json:"thread_name,omitempty"`
```

Comment:
A nitpick, but I think it would be better to put it before the flags, to avoid visual separation from other fields.